src/main/java/ui/screens/BurndownScreen.java

```diff
@@ -24,32 +45,75 @@ public BurndownScreen(ScreenManager screenManager, String id, String fxmlFilenam
     @Override
     protected void beforeVisualizationMount() {
         this.burndown = new Burndown();
+        checkComboBox.disableProperty().bind(this.burndown.serviceRunning());
     }
 
     @Override
     protected StackPane visualization() {
         return this.burndown;
     }
 
+    protected void updateDisplayedSprints(boolean overlay){
+        ArrayList<Sprint> list = new ArrayList<>(checkComboBox.getCheckModel().getCheckedItems());
+        Collections.reverse(list);
+        this.burndown.selectSprints(list, overlay);
+    }
+
+
     /**
      * Adds the switch sprint functionality to the sprint dropdown after the page is rendered.
      */
     @Override
     protected void afterVisualizationMount() {
-        sprint_combobox.setOnAction((e) -> {
-            Sprint sprint = sprint_combobox.getValue();
-            if (sprint == null) {
-                return;
-            }
-            this.burndown.switchSprint(sprint);
+        checkComboBox.getCheckModel().getCheckedItems().addListener(
+            (ListChangeListener<? super Sprint>) (change) -> {
+                if(checkComboBox.getCheckModel().getCheckedItems().size() > 1){
+                    overlayCheckBox.setVisible(true);
+                    overlayCheckBox.setDisable(false);
+                    overlayLabel.setVisible(true);
+                    updateDisplayedSprints(overlayCheckBox.isSelected());
+                }
+                else{
+                    overlayCheckBox.setVisible(false);
+                    overlayCheckBox.setDisable(true);
+                    overlayLabel.setVisible(false);
+                    updateDisplayedSprints(false);
+                }
+                Platform.runLater(() -> checkComboBox.hide());
+            });
+
+        overlayCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
+            updateDisplayedSprints(overlayCheckBox.isSelected());
         });
-        sprint_combobox.getSelectionModel().selectLast();
-        this.burndown.switchSprint(sprint_combobox.getValue());
+
+        updateDisplayedSprints(overlayCheckBox.isSelected());
+    }
+
+
+    @Override
+    protected void beforeParameterMount() {
```

Comment:
Having an issue where the combobox will display previous project sprints.

Reply:
should be fixed in latest commits, the commbobox was replaced entirely